enterprise_access/apps/content_assignments/models.py

```diff
@@ -234,30 +237,36 @@ def bulk_update(cls, assignment_records, updated_field_names):
             batch_size=BULK_OPERATION_BATCH_SIZE,
         )
 
-    def get_successful_linked_action(self):
+    def get_auto_expiration_date(self):
         """
-        Returns the first successful "linked" LearnerContentAssignmentActions for this assignment,
+        Returns the date at which this assignment expires due to
+        waiting too long to move into the "accepted" state.
+        """
+        last_notification = self.get_last_successful_notified_action()
+        if not last_notification:
+            return None
+        return last_notification.completed_at + timezone.timedelta(days=NUM_DAYS_BEFORE_AUTO_CANCELLATION)
```

Comment:
Can you document that triggering a reminder on an assignment does not re-set the auto-expiration timer?